packages/react-native-web/src/modules/usePlatformMethods/index.js

```diff
@@ -43,22 +44,30 @@ function setNativeProps(node, nativeProps, classList, pointerEvents, style, prev
  * Adds non-standard methods to the hode element. This is temporarily until an
  * API like `ReactNative.measure(hostRef, callback)` is added to React Native.
  */
-export default function usePlatformMethods(props: Object) {
+export default function usePlatformMethods({
+  classList,
+  pointerEvents,
+  style
+}: {
+  classList?: Array<string | boolean>,
+  style?: $PropertyType<ViewProps, 'style'>,
```

Comment:
This type can't be correct because components other than `View` set `style` too

Reply:
Good point, and I see they have different style props. OK, I made this generic now which is hand-wavy right. But because I used a star it's not going to really do anything.

As an aside, the type of this function doesn't really matter because we'd have to fix all the call points. Many of them use `pick` to get the props. And `pick` causes all type-safety to be lost because it uses Object. I think the correct signature is `T => $Shape<T>` or something like that. But that causes a whole bunch of issues which would need correcting (for instance because we later assign domProps to what would then be viewProps so it gets mad). So maybe an issue for another diff...